packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -143,4 +143,210 @@ class CupertinoIcons {
+  /// See also:
+  ///
+  ///  * [video_camera_solid], which is similar, but filled in.
+  ///  * [photo_camera], for the still photograph equivalent.
+  static const IconData video_camera = IconData(0xf4cc, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera for moving pictures. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [video_camera], which is similar, but not filled in.
+  ///  * [photo_camera_solid], for the still photograph equivalent.
+  static const IconData video_camera_solid = IconData(0xf4cd, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera containing two circular arrows pointing at each other, which indicate switching. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [switch_camera_solid], which is similar, but filled in.
+  static const IconData switch_camera = IconData(0xf49e, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera containing two circular arrows pointing at each other, which indicate switching. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [switch_camera], which is similar, but not filled in.
+  static const IconData switch_camera_solid = IconData(0xf49f, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A collection of envelopes (or folders), which store collections of files. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [collections_solid], which is similar, but filled in.
+  static const IconData collections = IconData(0xf3c9, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A collection of envelopes (or folders), which store collections of files. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [collections], which is similar, but not filled in.
+  static const IconData collections_solid = IconData(0xf3ca, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A single envelope (or folder), which stores multiple files. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [folder_solid], which is similar, but filled in.
+  ///  * [folder_open], which is the pre-iOS 7 version of this icon.
+  static const IconData folder = IconData(0xf434, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A single envelope (or folder), which stores multiple files. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [folder], which is similar, but not filled in.
+  ///  * [folder_open], which is the pre-iOS 7 version of this icon and not filled in.
+  static const IconData folder_solid = IconData(0xf435, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A single envelope that indicates being opened. A folder like this typically stores multiple files.
+  ///
+  /// See also:
+  ///
+  ///  * [folder], which is the equivalent of this icon for iOS versions later than or equal to iOS 7.
+  static const IconData folder_open = IconData(0xf38a, fontFamily: iconFont, fontPackage: iconFontPackage);
```

Comment:
doesn't quite look like an open folder. It just has thicker strokes. I wouldn't add this one

Reply:
It is a different version of `folder` and since I could not use the same name twice, I just [adopted the name from material/icons.dart, where the icon looks about identical](https://docs.flutter.io/flutter/material/Icons-class.html#folder_open).